doc/user/DESKTOP_INTEGRATION.md

```diff
@@ -2,6 +2,11 @@
 
 F3D can be integrated in the desktop experience.
 
+F3D now supports rendering in sandboxed environments. To enable thumbnail generation in sandboxed environments (e.g., Nautilus), add the `--rendering-backend` option to specify EGL as the backend:
+
+For Example:
+Exec=f3d --rendering-backend=egl --output=%o --resolution=%s,%s %i
```

Comment:
thats nice, but this is specific to linux, so you may want to put that in the Linux section